eclipse-scout-core/src/Extension.ts

```diff
@@ -43,9 +43,18 @@ export class Extension<E> {
     let origFunc = extended[funcName];
     let extension = this;
     extended[funcName] = function(...args) {
+      const extendedOrig = extension.extended;
+      const nextOrig = extension.next;
```

Comment:
Potential confusion with the word "orig" ("origFunc" vs. "nextOrig"), also inconsistent naming